lib/msal-angular/docs/multi-tenant.md

```diff
@@ -45,3 +45,39 @@ export class AppComponent implements OnInit {
     });
   }
 ```
+
+## Multi-tenant access tokens
```

Comment:
I think it would be good to include two sub sections here, one for the guard, and one for the interceptor.